public-docker-images/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.swp
```

Comment:
I don't think we need this? We already have a `.gitignore` for the project.

Reply:
This was just from the file I copied from your repo -- I'm not sure what it was there for?

Reply:
Ah, I was confused because it was a commend on the line -- thought you meant we don't need `.swp` ;)

I think this file just snuck in, removing it.